### ISSUE TITLE
`npm ls <pkgname>` always exits in error (even when there are no ls problems)

### ISSUE THREAD

**Issue by isaacs** [npm/cli#1979]
This is weird...

```
$ node . ls eslint
npm@7.0.2 /Users/isaacs/dev/npm/cli
├─┬ eslint-plugin-import@2.22.1
│ └── eslint@7.11.0 deduped
├─┬ eslint-plugin-node@11.1.0
│ ├─┬ eslint-plugin-es@3.0.1
│ │ └── eslint@7.11.0 deduped
│ └── eslint@7.11.0 deduped
├─┬ eslint-plugin-standard@4.0.1
│ └── eslint@7.11.0 deduped
└── eslint@7.11.0

$ echo $?
1
```

**Comment by ljharb** [comment]
I'm surprised that if `npm ls` exits nonzero, that some kind of warning wouldn't automatically be generated on `npm install`?

**Comment by nlf** [comment]
is this still an issue? i'm not able to reproduce it from `latest` on my end

**Comment by nlf** [comment]
this was fixed in `7.0.3`
